schemas/validate-all-tools.py

```diff
@@ -7,8 +7,8 @@
 import fastjsonschema
 
 
-HELP = """Usage: validate-all-tools.py -s <SCHEMA_DEFINITION_FILE> -t <TOOLS_DIRECTORY>"""
-SCHEMA_DEFINITION_FILE = None
+HELP = """Usage: validate-all-tools.py -s <SCHEMA_DEFINITION_FILE1> -t <TOOLS_DIRECTORY>"""
```

Comment:
A suggestion could be to allow -s to load an entire directory of schemas adding the option of passing multiple schemas via a single parameter

Reply:
Yes, but I don't think we're going to have too many schemas and that sounds like too much work :) I'd say it's ok as it is now.